cmd/test/main.go

```diff
@@ -106,6 +119,22 @@ func test(ctx *TestCommandContext, paths []string, flags TestCommandFlags) error
 		return err
 	}
 
+	if paths[0] == "-" {
+		if len(paths) > 1 {
+			return fmt.Errorf(fmt.Sprintf("Unexpected args: [%s]", strings.Join(paths[1:], ",")))
+		}
+		tempFile, err := ioutil.TempFile("", "datree_temp_*.yaml")
```

Comment:
you should generate a new name for each run, maybe using the date, in case the process will die before deleting the temp file so the next run will not get affected

Reply:
the `*` in the second argument does exactly that :)